tests/casp/oci_install.pm

```diff
@@ -21,7 +21,7 @@ sub run() {
     send_key 'alt-p';    # partitioning
     assert_screen 'prepare-hard-disk';
     send_key 'alt-b';    # back
-    if (check_screen 'error-small-disk') {
+    if (check_screen 'error-small-disk', 3) {
```

Comment:
I recommend to use check_screen only with timeout of 0 or not at all. I consider a multi-tag assert_screen with use of `match_has_tag` a better alternative.

So either

```perl
assert_screen [qw(oci-overview-filled error-small-disk)];
if (match_has_tag 'error-small-disk')) {
    record_soft_failure …;
    send_key 'ret';
    assert_screen 'oci-overview-filled';
}
```

or

```perl
wait_screen_change { send_key 'alt-b' };
check_screen 'error-small-disk', 0;
if (…)
```

This way we don't have to do the dangerous dice-throwing game on the timeout value between "waste time waiting for an error which normally does not appear" and "sporadically failing job because the error message takes a bit longer to show up, e.g. 4 seconds"